job/job.go

```diff
@@ -86,3 +89,20 @@ func GetGenericJob(obj Object, config Config) (*batchv1.Job, error) {
 
 	return job, err
 }
+
+func getK8upResourceLimits(r *corev1.ResourceRequirements) corev1.ResourceRequirements {
+	if r == nil {
```

Comment:
Maybe we can do this check more granularly?

Given:
* global memory limit :1Gb
* global cpu limit: 1000m

Now if the user creates a job or schedule with cpu limit 1500m an no memory limit, no memory limit will be set at all. In that case it would make more sense to apply the global default to these nil values.